docs/cloud/rahti2/tutorials/index.md

```diff
@@ -1,27 +1,21 @@
 * [Accessing databases on Rahti 2 from CSC supercomputers](connect-database-hpc.md)
+* [Advanced NetworkPolicies](advanced-NetworkPolicies-networking.md)
+* [Allow pulls of images from one Rahti project to another](allow-pull-from-other-projects.md)
 * [Annotations](annotations.md)
 * [CI/CD on Rahti 2](ci_cd_introduction.md)
 * [Configuration, ConfigMaps and Secrets](configuration.md)
 * [Custom domain names and secure transport](custom-domain.md)
-* [Deploying a static web server using the command line](elemental_tutorial.md)
-* [Deploying a static web server using the web interface](basic-console.md)
-* [How to add docker hub credentials to a project](docker_hub_login.md)
-* [How to automatically scale up and down replicas](addHorizontalAutoscaler.md)
-* [How to backup a Postgres DB into Allas](backup-postgres-allas.md)
-* [How to package a Kubernetes application with Helm](helm.md)
-* [How to run an ad-hoc interactive container](oc-run.md)
-* [How to transfer data to Rahti 2?](transfer_data_rahti.md)
+* [Deploying a static web server using the command line](deploy_static_webserver_cli.md)
+* [Deploying a static web server using the web interface](deploy_static_webserver_webconsole.md)
+* [Integrating External Services](integrating-external-services.md)
 * [Kustomize](kustomize.md)
 * [Learn cloud computing by developing and deploying a web application](web-app-dev-cloud.md)
 * [Multi stage builds](multi-stage-builds.md)
+* [Pod (anti) affinity](pod-affinity.md)
 * [Reverse proxy authentication using a sidecar container](sidecar_proxy_authentication.md)
 * [Sending e-mail from Rahti 2](email.md)
+* [Setup a HTTP redirection in Rahti](http-redirector.md)
 * [Short introduction to YAML](yaml_introduction.md)
-* [SSH Jumphost in cPouta](create-jumphost.md)
-* [SSH Key-pair](ssh-key.md)
 * [Webhooks](webhooks.md)
-* [Pod (anti) affinity](pod-affinity.md)
-* [Advanced NetworkPolicies](advanced-NetworkPolicies-networking.md)
-* [Integrating External Services](integrating-external-services.md)
-* [Setup a HTTP redirection in Rahti](http-redirector.md)
-* [Allow pulls of images from one Rahti project to another](allow-pull-from-other-projects.md)
+
+We also have a [Rahti 2 FAQ](../../../support/faq#rahti-2). Take a look at it.
```

Comment:
Suggestion: Change the sentence. 

We also have an [FAQ](../../../support/faq#rahti-2) page for Rahti. Take a look at it!

Reply:
There is a button in Github for suggestion comments:
![Screenshot from 2024-11-28 15-57-53](https://github.com/user-attachments/assets/868e8fc9-0764-4117-8a14-29d71497415f)

Reply:
Also your suggestion has a typo, it is `a FAQ`.

Reply:
Thank you Alvaro for pointing that out. I will use that button. 
About FAQ, I think the right way is an FAQ since the first letter sounds like EF. https://www.quora.com/Which-is-correct-a-FAQ-or-an-FAQ

Reply:
ups, you are right  😅